src/World.js

```diff
@@ -36,6 +38,26 @@ class World {
                 })
             })
     }
+
+    segmentResults(eventSubgroupId) {
+        return this.request.protobuf(`/api/segment-results?world_id=0&segment_id=1&event_subgroup_id=${eventSubgroupId}&full=true&player_id=0`)
```

Comment:
I guess `world_id`, `segment_id` and `player_id` are just ignored. Does it still work if those parameters are left out?

`world_id` is actually passed into the constructor, so possibly should be `world_id=${this.worldId}` to be consistent (even though it's always 1 and probably ignored).